dotCMS/src/main/java/com/dotcms/api/system/event/local/OrphanEvent.java

```diff
@@ -0,0 +1,28 @@
+package com.dotcms.api.system.event.local;
+
+import java.io.Serializable;
+
+/**
+ * This is a wrapper event for orphan (events with not subscribed associated)
+ * @author jsanca
+ */
+public class OrphanEvent implements Serializable {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a "static final long serialVersionUID" field to this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2057)